common/src/main/scala/io/prediction/authentication/KeyAuthentication.scala

```diff
@@ -0,0 +1,45 @@
+package io.prediction.authentication
+
+/**
+  * This is a (very) simple authentication for the dashboard and engine servers
+  * It is highly recommended to implement a stonger authentication mechanism
+  */
+
+import java.io.File
+
+import com.typesafe.config.ConfigFactory
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import spray.http.HttpRequest
+import spray.routing.{AuthenticationFailedRejection, RequestContext}
+import spray.routing.authentication._
+import spray.routing.directives.AuthMagnet
+import scala.concurrent.Future
+
+
+trait KeyAuthentication {
+
+  object ServerKey {
+    val serverConfig = ConfigFactory.parseFile(new File("conf/server.conf"))
```

Comment:
This expects the configuration file to live relative to where the `pio` command is run. How about using this method? (http://typesafehub.github.io/config/latest/api/com/typesafe/config/ConfigFactory.html#load-java.lang.String-)